pkg/services/login/model.go

```diff
@@ -67,7 +67,12 @@ type ExternalUserInfo struct {
 }
 
 func (e *ExternalUserInfo) String() string {
-	return fmt.Sprintf("%+v", *e)
+	isGrafanaAdmin := "nil"
+	if e.IsGrafanaAdmin != nil {
+		isGrafanaAdmin = fmt.Sprintf("%v", *e.IsGrafanaAdmin)
+	}
+	return fmt.Sprintf("OAuthToken: %+v, AuthModule: %v, AuthId: %v, UserId: %v, Email: %v, Login: %v, Name: %v, Groups: %v, OrgRoles: %v, IsGrafanaAdmin: %v, IsDisabled: %v, SkipTeamSync: %v",
```

Comment:
This might be kinda hard to read. I don't have a better solution but I'd prefer a line per value, rather than a really long string.

Reply:
This is for the logs, where new lines are discouraged, is that what you meant?

Reply:
Yes! I'm aware it's used for a log. I stand by my comment. Keep in mind I don't have a better solution, so feel free to merge as is.